src/screens/CreateCommunity/index.js

```diff
@@ -200,6 +209,7 @@ const CreateCommunity = () => {
             <View style={styles.containerInput}>
               <View style={{flex: 1}}>
                 <Input
+                  ref={inputRef}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Passing the ref to the `Input` component is correct. This will allow the parent component to access the underlying `TextInput` component.
```javascript
ref={inputRef}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->